test/kwok/kwok.test.sh

```diff
@@ -54,6 +54,32 @@ function test_node_ready() {
   fi
 }
 
+# Check pod status
+function check_pod_status() {
+  for row in $(kubectl get po -ojson | jq -r '.items[] | @base64'); do
+      _jq() {
+      echo ${row} | base64 --decode | jq -r ${1}
+      }
+    podName=$(_jq '.metadata.name')
```

Comment:
```suggestion
    pod_name=$(_jq '.metadata.name')
```

At bash we generally use the snake case

Reply:
Made changes [here](https://github.com/kubernetes-sigs/kwok/compare/42dd2e5d7ed0d857067ceb1187a1e5f39b37c4fa..6937431a58b0a2149d9fe50882102895d9b55002). Thanks.